extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/ConfigurationService.java

```diff
@@ -206,4 +206,22 @@ public EncryptionMethod getEncryptionMethod() throws GuacamoleException {
         );
     }
 
+    /**
+     * Returns Max results a ldap query can return
```

Comment:
It looks like maybe a few words were left out here? Did you mean to say "return, as specified with guacamole.properties"?